components/contribution-flow/ContributionFlowHeader.js

```diff
@@ -57,7 +58,7 @@ class NewContributionFlowHeader extends React.Component {
               />
             </H1>
           </CollectiveTitleContainer>
-          {contributors?.length > 0 && (
+          {contributors?.length > 0 && !isCrypto && (
```

Comment:
I don't see this as necessary. Why was this added?

Reply:
@znarf : I wanted to hide the contributor avatars. If we do want to show the contributors avatars we need to differentiate between custom contributors vs crypto contributors. We don't have that support yet. My thinking is not to do this on the MVP. Let me know if you think otherwise. 🤔

Reply:
If contributors is empty that's not necessary.

Reply:
Ah yes, that makes sense. I've corrected that.